### DISCUSSION TITLE
Skipped *.*.*.*:88 from target list as found unresponsive 32 times. tell me why?

### DISCUSSION THREAD

**Discussion by wlaq-su** [projectdiscovery/nuclei#4524] [Q&A] [answered]
[INF] Skipped ip:88 from target list as found unresponsive 32 times tell me why?

**Comment by ehsandeep** [comment]
```
nuclei -h mhe

   -mhe, -max-host-error int  max errors for a host before skipping from scan (default 30)
```

**Reply by wlaq-su** [reply]
Does the template name support Chinese?

**Comment by wlaq-su** [comment]
> ```
> nuclei -h mhe
> 
>    -mhe, -max-host-error int  max errors for a host before skipping from scan (default 30)
> ```

What command to execute to solve it ？

**Reply by ehsandeep** [reply, marked as answer]
if you don't want to skip unresponsive host, you can use below option

```
   -nmhe, -no-mhe                   disable skipping host from scan based on errors
```

**Reply by wlaq-su** [reply]
thanks mogul
